.cicd/helpers/execute.sh

```diff
@@ -0,0 +1,4 @@
+function execute() {
+  echo "--- Executing: $@"
```

Comment:
When all folds are expanded I need something clear to search for with command+f.

Reply:
> When all folds are expanded I need something to search for with command+f
> Same for in Travis

All folds aren't expanded.   

> `echo "--- Executing: $@"`

All folds are collapsed, so we don't need to [Ctrl] + [F].

Reply:
Remember the button Eric told you about to expand them all? Super handy